tests/test_minhash.py

```diff
@@ -1675,10 +1675,27 @@ def test_intersection_1_num():
     mh2.add_hash(2)
 
     mh3 = mh1.intersection(mh2)
-    print(set(mh3.hashes))
+    print("mh.intersection INTERSECTION HASHES:",set(mh3.hashes))
     assert len(mh3) == 1
     assert 0 in mh3.hashes
 
+def test_and_operator():
+    mh1 = MinHash(20, 21)
+    mh1.add_hash(5)
+    mh1.add_hash(6)  
```

Comment:
```suggestion
    mh1.add_hash(6)
```
there appears to be an unnecessary space at the end of this line, so suggesting a removal - you can commit the suggestion.